src/main/java/com/ticketbooking/dto/MovieDto.java

```diff
@@ -0,0 +1,35 @@
+package com.ticketbooking.dto;
+
+import java.time.LocalDateTime;
+
+public class MovieDto {
+    private Long movieId;
+    private Long screeningId;
```

Comment:
Zwracasz w endpoincie filmów screeningi, czy to na pewno dobrze?

Reply:
Tym screeningId posługuję się przy wyświetleniu dostępnych miejsc na sali a potem do dokonania rezerwacji